components/resource_control/src/channel.rs

```diff
@@ -152,6 +152,8 @@ impl<T: Send + 'static> Sender<T> {
     }
 }
 
+unsafe impl<T: Send + 'static> Sync for Sender<T> {}
```

Comment:
Forcefully making RefCell `last_msg_group` to be sync will cause UB. And sharing the field is probably not what the priority scheduling expects too. cc @Connor1996

Reply:
yes, shouldn't impl sync for sender

Reply:
Thx, it has been revoked.